src/pheval/analyse/analysis.py

```diff
@@ -368,19 +500,46 @@ def _assess_phenopacket_variant_prioritisation(
     ).assess_variant_prioritisation(variant_rank_stats, variant_rank_comparison)
 
 
+def _assess_phenopacket_disease_prioritisation(
+    standardised_disease_result: Path,
+    score_order: str,
+    results_dir_and_input: TrackInputOutputDirectories,
+    threshold: float,
+    disease_rank_stats: RankStats,
+    disease_rank_comparison: defaultdict,
+) -> None:
+    """Assess gene prioritisation for a phenopacket."""
+    phenopacket_path = obtain_closest_file_name(
+        standardised_disease_result, all_files(results_dir_and_input.phenopacket_dir)
+    )
+    pheval_disease_result = _read_standardised_result(standardised_disease_result)
+    proband_diseases = _obtain_causative_diseases(phenopacket_path)
+    AssessDiseasePrioritisation(
+        phenopacket_path,
+        results_dir_and_input.results_dir.joinpath("pheval_disease_results/"),
+        parse_pheval_disease_result(pheval_disease_result),
+        threshold,
+        score_order,
+        proband_diseases,
+    ).assess_disease_prioritisation(disease_rank_stats, disease_rank_comparison)
+
+
 def _assess_prioritisation_for_results_directory(
     results_directory_and_input: TrackInputOutputDirectories,
     score_order: str,
     threshold: float,
     gene_rank_comparison: defaultdict,
     variant_rank_comparison: defaultdict,
+    disease_rank_comparison: defaultdict,
     gene_stats_writer: RankStatsWriter,
     variants_stats_writer: RankStatsWriter,
+    disease_stats_writer: RankStatsWriter,
     gene_analysis: bool,
     variant_analysis: bool,
+    disease_analysis: bool,
 ) -> TrackPrioritisation:
     """Assess prioritisation for a single results directory."""
-    gene_rank_stats, variant_rank_stats = RankStats(), RankStats()
+    gene_rank_stats, variant_rank_stats, disease_rank_stats = RankStats(), RankStats(), RankStats()
```

Comment:
I have never seen this syntax before, what does it mean to have two function calls separated by comma generating three results?

Reply:
This is the same as

```python
gene_rank_stats = RankStats()
variant_rank_stats = RankStats()
disease_rank_stats = RankStats()
```

I just write it like this in one line to save lines of code

Reply:
Thank you!